jobs/cloud_controller_clock/templates/blobstore_configs.json.erb

```diff
@@ -0,0 +1,54 @@
+<%
+require "json"
+
+def cli_cfg_with_default_timeout(connection_cfg, blobstore_type, default_seconds: 41)
+  cfg = (connection_cfg || {}).dup
+  if blobstore_type == 'storage_cli'
+    if !cfg.key?('put_timeout_in_seconds') || cfg['put_timeout_in_seconds'].to_s.empty?
+      cfg['put_timeout_in_seconds'] = default_seconds.to_s
+    end
+  end
+  cfg
+end
+
+def add(h, key, val)
+  return if val.nil?
+  return if val.respond_to?(:empty?) && val.empty?
+  h[key] = val
+end
+
+def options_for(scope_prefix, provider_prop)
+   provider = p(provider_prop)
+  if provider != "AzureRM"
```

Comment:
The name "AzureRM" comes from the Fog implementation. Shouldn't we use another name?